resident/resident-service/src/main/java/io/mosip/resident/service/impl/BaseWebSubInitializer.java

```diff
@@ -77,16 +80,11 @@ public void onApplicationEvent(ApplicationReadyEvent applicationReadyEvent) {
 
     }
 
-    private void authTransactionSubscription() {
-        try{
-            subscribe(authTransactionTopic, callbackAuthTransactionUrl, authTransactionSecret, hubUrl);
-        }
-        catch (Exception e){
-            logger.error( "authTransactionSubscription", "BaseWebSubInitializer", "Exception while subscribing to topic: " + e.getMessage());
-        }
+    public void authTransactionSubscription() {
+        subscribe(authTransactionTopic, callbackAuthTransactionUrl, authTransactionSecret, hubUrl);
     }
 
-    protected void tryRegisterTopicEvent(String eventTopic) {
+    public void tryRegisterTopicEvent(String eventTopic) {
```

Comment:
Do you need public modifier here? can it remain protected?

Reply:
No changed it to protected.